src/features/notifications/NotificationListItem.tsx

```diff
@@ -29,16 +29,20 @@ const NotificationListItem = ({
 
   const time = useMemo(() => {
     const date = new Date(notification.time)
+    // To local time
+    date.setMinutes(date.getMinutes() - date.getTimezoneOffset())
     // Get hours with leading zero
-    const hours = date.getHours().toString().padStart(2, '0')
+    const hours = (date.getHours() % 12 || 12).toString().padStart(2, '0')
```

Comment:
I wonder if there's a utility from `date-fns` that could help here?

Reply:
Let me check